gpu-operator/platform-support.rst

```diff
@@ -539,7 +538,11 @@ KubeVirt and OpenShift Virtualization with NVIDIA vGPU is supported on the follo
 
 Note that HGX platforms are not supported.
 
-Support for GPUDirect RDMA
+.. note::
+  
+  KubeVirt with NVIDIA vGPU is supported on ``nodes`` with Linux kernel < 6.0, such as Ubuntu 22.04 ``LTS``.
```

Comment:
do we need to say ``nodes`` here in the formatting? just curious.

otherwise lgtm

Reply:
Yeah, that should be correct.